extensions/wikia/CookiePolicy/scripts/cookiePolicy.js

```diff
@@ -0,0 +1,69 @@
+/* global GlobalNotification */
+require([
+	'jquery',
+	'mw',
+	'wikia.window',
+	'wikia.cookies',
+	'wikia.geo'
+], function ($, mw, window, cookies, geo) {
+	'use strict';
+
+	/**
+	 * Initialize JS for whether or not to show a banner notifying European Union users that we use cookies
+	 * on our site, per EU law. Users can dismiss the notification and will not be shown again, unless
+	 * they clear their cookies.
+	 */
+	function initCookieNotification() {
+		if (shouldShowBanner()) {
+			showBanner();
+		}
+	}
+
+	/**
+	 * Check if user is from the EU and hasn't seen the banner yet
+	 * @returns {boolean}
+	 */
+	function shouldShowBanner() {
+		var fromEU = geo.getContinentCode() === 'EU';
+
+		return (fromEU && !hasSeenBanner());
+	}
+
+	/**
+	 * Display the cookie policy message to the user
+	 */
+	function showBanner() {
+		var message = mw.message('cookie-policy-notification-message').parse();
+
+		GlobalNotification.show(message, 'notify');
```

Comment:
@adamkarminski, currently, Global Notifications don't support two instances, but we'll add that support with https://wikia-inc.atlassian.net/browse/UC-174. We won't actually release this cookie policy code until that ticket is done in the next sprint. (https://github.com/Wikia/config/pull/868/files disables this extension for now)